private-templates-service/client/src/components/Common/ShareModal/ShareModal.tsx

```diff
@@ -35,4 +70,14 @@ class ShareModal extends React.Component<ShareModalProps> {
   }
 }
 
-export default ShareModal;
\ No newline at end of file
+function onCopy(props: ShareModalProps) {
```

Comment:
`onCopy` is the method invoked when user wants to copy the shareable URL from the ShareModal. On second thought I'll rename it to `onCopyURL` so that it's more clear.